Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -297,6 +321,22 @@ open class NavigationMapView: UIView {
         updateRestrictedAreasGradientStops(along: nil)
     }
     
+    func removeAlternativeRoutes() {
```

Comment:
Is it duplication from `Sources/MapboxNavigation/NavigationMapView+ContinuousAlternatives.swift`?

Reply:
No. Alternative routes is all other routes from the used RouteResponse. Continuous alternatives are the routes which were detected during navigation sessions. These are different

Reply:
But can we reuse the code? The only diff is the routes array: 
`routes?.dropFirst()` vs `continuousAlternatives?.compactMap(\.indexedRouteResponse.currentRoute)`

Reply:
It would be reasonable to unify `removeRoutes()`, `removeAlternativeRoutes(), `removeRestrictedRouteArea() and `removeContinuousAlternativesRoutesLayers()` somehow, but they are all a bit different here and there which makes extracting the common part less profitable. Some use enumared routes, some are not; some use different sets of identifiers to operate. Having this configurable adds and overhead which is comparable to the current implementation IMHO